recipes/centos_jdk8/contrib/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2012-2017 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+# Contributors:
+# Red Hat, Inc. - initial implementation
+
+set -e
+
+# insert settings for mirrors/repository managers into settings.xml if supplied
+configure_mirrors() {
+  echo "Setting maven mirror from $MAVEN_MIRROR_URL"
```

Comment:
`MAVEN_MIRROR_URL` may be not set here and you would log:

```
Setting maven mirror from
```
You may want to move this log inside the `if` clause body.

Reply:
oops!!! sorry @l0rd  - i was having that for test missed to removed it before i pushed